CMakeLists.txt

```diff
@@ -31,11 +31,13 @@ if(OPTIONAL26_ENABLE_TESTING)
     FetchContent_MakeAvailable(googletest)
 endif()
 
+set(CMAKE_VERIFY_INTERFACE_HEADER_SETS ON)
+
 # Create the library target and named header set for beman_optional26
-add_library(beman_optional26 STATIC)
+add_library(beman_optional26 INTERFACE)
```

Comment:
```suggestion
add_library(beman.optional26 INTERFACE)
```
The standard says it should use `.` after beman. Would you like to apply these updates in your PR?

<img width="1033" alt="image" src="https://github.com/user-attachments/assets/4474122d-56e8-4da2-bfef-07f342474d63">

Reply:
I can fix this up as part of the PR. Need to merge in other changes in any case.

Reply:
That would be great. But feel free to speed-up/skip steps today if you need something in main for Poland.

Reply:
Note: that will end in very strange behaviour! Normally I use something like this:
```cmake
find_package(Boost CONFIG QUIET)
if(TARGET Boost::headers AND TARGET fmt::fmt)
    add_executable(to_string to_string.cpp)
    target_link_libraries(to_string fmt::fmt Boost::headers)
endif()
```

Reply:
This is creating the target that gets the named header file_set added to it? It's up here to avoid ordering problems between subdirectories.